be/src/vec/runtime/vdatetime_value.h

```diff
@@ -235,6 +235,7 @@ class VecDateTimeValue { // Now this type is a temp solution with little changes
     VecDateTimeValue()
             : _neg(0),
               _type(TIME_DATETIME),
+              _microsecond(0),
```

Comment:
We should not support microsecond for VecDateTimeValue. We only need this for DateV2

Reply:
removed it